packages/run/.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
please remove this file. the `.gitignore` at the root of the repo will handle this